### ISSUE TITLE
VictorSP Set Not Accurate

### ISSUE THREAD

**Issue by wvauclain** [Open-RIO/ToastAPI#38]
Whenever I use a VictorSP, at least in sim, the motor controller gets set to what it needs to be minus 0.05, so if I were to simply access both of them in the registrar, on the sim RoboRIO their signal immediately drops to -0.05. You can sets them to -1 just fine, but the maximum signal strength a VictorSP can be set to is 0.94.

**Comment by LoadingPleaseWait** [comment]
I can confirm that this bug is in Toast for me too.

**Comment by JaciBrunning** [comment]
This is due to each Motor Controller having it's own PWM timings. What we do is we patch PWM, not the Motor Controller. We times this by a rough value to get approximately what each controller is running at within a reasonable margin of error. It's not exact, but it's the closest we can get without patching 'front-end classes'.

**Comment by wvauclain** [comment]
So this means that the error is simply a problem with the simulator, and will not be present on the RoboRIO itself?

**Comment by JaciBrunning** [comment]
Correct. Realistically, a difference that small would not cause an issue,
as it is too small for most motors / controllers

**Comment by Samcfuchs** [comment]
Is this issue closed now?
